lib/client.js

```diff
@@ -367,12 +342,10 @@ exports.createClient = function(reference, name) {
             }
             debug("Request error calling %s NOT retrying!, err: %s, JSON: %s",
                   entry.name, err, err);
-            if (reportStats) {
-              var errCode = err.code || err.name;
-              if (!errCode || typeof(errCode) !== 'string') {
-                errCode = 'Error';
-              }
-              reportStats(false, errCode);
+            if (monitor) {
+              var d = process.hrtime(start);
+              monitor.measure(entry.name, d[0] * 1000 + d[1] / 1000000);
+              monitor.count(entry.name);
```

Comment:
Can we do: `monitor.measure([entry.name, 'server-error'], ...` or `monitor.measure([entry.name, 'client-error'], ...`

depending on whether it's a 4xx or 5xx error...